prebid-mobile/adops-setup-full-screen-video-mopub.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page_v2
+title: Setup Rewarded Video for MoPub
+description: Setup Full Screen Video for MoPub
+pid: 1
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile-adops
+sidebarType: 3
+---
+
+# Step-by-Step Line Item Setup for Full-screen Video on MoPub
+
+* TOC
+{:toc }
+
+This page describes step-by-step instructions to set up full-screen video line items on MoPub to be used with the Prebid Mobile SDK.
+
+## Step 1. Create full screen adUnit  
+<img src="/assets/images/ad-ops/mopub/full/mopub-lineitem-full-adunit.png">&nbsp;&nbsp;
```

Comment:
Would be best to always have some written description, even when the picture says it all, for accessibility.